sample/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'com.android.library'
```

Comment:
You honestly might be able to remove the sample app and just stick with the README/wiki examples. The reason I did this was because it was the only way I could get android studio to properly work with this and test against. Otherwise it treats this as just an external gradle project and doesn't really support most of the normal perks of Android Studio (project syncing, debugger, etc).